dd-sdk-android/src/main/kotlin/com/datadog/android/rum/GlobalRum.kt

```diff
@@ -107,6 +113,15 @@ object GlobalRum {
 
     // region Internal
 
+    internal fun addUserInteraction() {
+        val nanoTime = System.nanoTime()
+        val isInactiveSession = (nanoTime - lastUserInteractionNs.get()) >= SESSION_INACTIVITY_NS
```

Comment:
you can extract this in a new method as it is being used also in the other function